components/about_build_modal/__snapshots__/about_build_modal.test.jsx.snap

```diff
@@ -340,17 +373,50 @@ exports[`components/AboutBuildModal should match snapshot for team edition 1`] =
         </a>
       </div>
       <div
-        className="form-group about-modal__copyright"
+        className="form-group"
       >
-        <FormattedMessage
-          defaultMessage="Copyright 2015 - {currentYear} Mattermost, Inc. All rights reserved"
-          id="about.copyright"
-          values={
-            Object {
-              "currentYear": 2017,
+        <div
+          className="about-modal__copyright"
+        >
+          <FormattedMessage
+            defaultMessage="Copyright 2015 - {currentYear} Mattermost, Inc. All rights reserved"
+            id="about.copyright"
+            values={
+              Object {
+                "currentYear": 2017,
+              }
             }
-          }
-        />
+          />
+        </div>
+        <div
+          className="about-modal__links"
+        >
+          <a
+            href="http://about.mattermost.com/default-terms/"
```

Comment:
This wasn't noted in the original ticket, my mistake. But instead of hard-coding the value, we should use the value set in SupportSettings.TermsOfServiceLink in config.json

Reply:
I've changed it to use the value from the config, but the reason the link in the snapshot is still the URL is because in the test setup, I set the config value to be the link.